src/test/java/uk/gov/justice/laa/crime/cfecrime/utils/RequestTestUtil.java

```diff
@@ -27,4 +40,61 @@ public static void setSectionPassportBenefit(CfeCrimeRequest request, boolean va
         sectionPassportedBenefit.withPassportedBenefit(Boolean.valueOf(value));
         request.withSectionPassportedBenefit(sectionPassportedBenefit);
     }
+
+    public static void setSectionInitMeansTest(CfeCrimeRequest cfeCrimeRequest, CaseType caseType, MagCourtOutcome magCourtOutcome)  {
+        List<DependantChild> dcList = new ArrayList<DependantChild>();
+        DependantChild dc = new DependantChild();
+        dc.setAgeRange(AgeRange.SIXTEEN_TO_EIGHTEEN);
+        dcList.add(dc);
+        Income  income = new Income();
+
+        SectionInitialMeansTest sectionInitialMeansTest = new SectionInitialMeansTest();
+        sectionInitialMeansTest.setCaseType(caseType);
+        sectionInitialMeansTest.setDependantChildren(dcList);
+        sectionInitialMeansTest.setHasPartner(false);
+        sectionInitialMeansTest.setIncome(income);
```

Comment:
bit confused by this too -it should be a list with types and values?

Reply:
same as above.